setup.py

```diff
@@ -18,10 +29,62 @@ def get_package_data(package, base_dir):
         for dirname in dirnames:
             get_package_data(package, dirname)
 
-get_package_data('nsot', 'nsot/static')
+get_package_data('nsot', 'nsot/static/build')
 get_package_data('nsot', 'nsot/templates')
 get_package_data('nsot', 'nsot/migrations')
 
+
+class PyTest(TestCommand):
+    user_options = [('pytest-args=', 'a', 'Arguments to pass to py.test')]
+
+    def initialize_options(self):
+        TestCommand.initialize_options(self)
+        self.pytest_args = []
+
+    def finalize_options(self):
+        TestCommand.finalize_options(self)
+        self.test_args = []
+        self.test_suite = True
+
+    def run_tests(self):
+        #import here, cause outside the eggs aren't loaded
+        log.info('Running python tests...')
+        import pytest
+        errno = pytest.main(self.pytest_args)
+        sys.exit(errno)
+
+
+class DevelopWithBuildStatic(DevelopCommand):
+    def install_for_development(self):
+        self.run_command('build_static')
+        return DevelopCommand.install_for_development(self)
+
+
+class SDistWithBuildStatic(SDistCommand):
+    def make_distribution(self):
+        self.run_command('build_static')
+        return SDistCommand.make_distribution(self)
+
+
+class BuildStatic(Command):
+    def initialize_options(self):
```

Comment:
Can you please add `user_options = []` so this can be run individually from the CLI?